app/containers/InterventionPage/ShareFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from 'react';
+import { Col } from 'react-grid-system';
+
+import globalMessages from 'global/i18n/globalMessages';
+
+import Select from 'components/Select';
+import { IntlShape } from 'react-intl';
+import { SelectOption } from 'components/Select/types';
+
+type Props = {
+  formatMessage: IntlShape['formatMessage'];
+  onChange: (values: string) => void;
+  active: string;
+};
+
+const ShareFilter = ({ formatMessage, onChange, active }: Props) => {
+  const options = useMemo(
+    () =>
+      Object.keys(globalMessages.sharingFilters).map((status) => ({
```

Comment:
Having enum mentioned above we could go with

```suggestion
  const options: SelectOption<SharingFilter>[] = useMemo(
    () =>
      Object.values(SharingFilter).map((status) => ({
```